spec/models/miq_provision_virt_workflow_spec.rb

```diff
@@ -84,24 +86,30 @@
         s11 = FactoryGirl.create(:switch, :name => "A")
         s12 = FactoryGirl.create(:switch, :name => "B")
         s13 = FactoryGirl.create(:switch, :name => "C")
-        @src_vm.host.switches = [s11, s12, s13]
+        s14 = FactoryGirl.create(:switch, :name => "D")
+        @src_vm.host.switches   = [s11, s12, s13]
+        @other_vm.host.switches = [s14]
         @lan11 = FactoryGirl.create(:lan, :name => "lan_A", :switch_id => s11.id)
         @lan12 = FactoryGirl.create(:lan, :name => "lan_B", :switch_id => s12.id)
         @lan13 = FactoryGirl.create(:lan, :name => "lan_C", :switch_id => s13.id)
+        @lan14 = FactoryGirl.create(:lan, :name => "lan_D", :switch_id => s14.id)
```

Comment:
@NickLaMuro - Were the `@lan14` and `@other_vm` additions to shore up the spec test cases or for another reason?